src/services/calculator-process.js

```diff
@@ -0,0 +1,56 @@
+let memory = [];
+function calculatorProcess(content,visor) {
+
+	const numbersOperators = (operator) => {
+		return String(
+			eval(
+				(memory[0]+memory[1])
```

Comment:
Hey @Willian1661 

why you get element 0 and 1 as a first parameter?

Reply:
i was thinking how to not get this.I created a test with a code sandbox and when i create an array and i pass the elements through the parameter(`OP`) i get the string, like this: `
let memory = ["10","10"]
const blabla = (op) => eval(memory[0] + op + memory[1])
let result = blabla("+")`
i get normally the first string
but the way is in the code i get like a **"char"** just the first character of the string

Reply:
> i was thinking how to not get this.I created a test with a code sandbox and when i create an array and i pass the elements through the parameter(`OP`) i get the string, like this: 
`let memory = ["10","10"] const blabla = (op) => eval(memory[0] + op + memory[1])
 let result = blabla("+")` 

i get normally the first string, but the way is in the code i get like a **"char"** just the first character of the string

Reply:
yep then it won't work!

should be good you don't get any code from web, try to provide your own solution!

Reply:
it's strange, i'm sending a string and receiving a char

Reply:
it solved

Reply:
> yep then it won't work!
> 
> should be good you don't get any code from web, try to provide your own solution!

and i did :p